src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -105,24 +105,24 @@ void printJournalFileMeta(bsl::ostream&                    ostream,
                           const mqbs::JournalFileIterator* journalFile_p,
                           bslma::Allocator*                allocator)
 {
-    BSLS_ASSERT_SAFE(journalFile_p && journalFile_p->isValid());
+    if (!journalFile_p || !journalFile_p->isValid()) {
+        return;  // RETURN
+    }
 
-    const bsl::vector<const char*> fields = {"BlazingMQ File Header",
+    const bsl::vector<bsl::string> fields = {"BlazingMQ File Header",
                                              "Journal File Header",
                                              "Journal SyncPoint"};
 
-    PRINTER_TYPE1 printer(ostream, &fields);
+    PRINTER_TYPE1 printer(ostream, fields);
     {
         bmqu::MemOutStream s(allocator);
-        s << '\n';
         mqbs::FileStoreProtocolPrinter::printFileHeader<PRINTER_TYPE2>(
             s,
             *journalFile_p->mappedFileDescriptor());
         printer << s.str();
     }
     {
         bmqu::MemOutStream s(allocator);
-        s << '\n';
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.